src/style.css

```
@@ -0,0 +1,36 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 :nth-child(-n+1) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--5 :nth-child(-n+5) {
+  background-image: url("images/star-active.svg");
+}
```

You can combine it like this:
```suggestion
.stars--1 :nth-child(-n+1),
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4),
.stars--5 :nth-child(-n+5) {
  background-image: url("images/star-active.svg");
}
```

---

Ok, understood